--- LoE.Shared.il.original	Fri Jan 30 14:32:07 2015
+++ LoE.Shared.il	Fri Jan 30 14:32:51 2015
@@ -7216,7 +7216,7 @@
       IL_0034:  ldloc.1
       IL_0035:  call       valuetype [mscorlib]System.DateTime [mscorlib]System.IO.File::GetLastWriteTimeUtc(string)
       IL_003a:  stloc.2
-      IL_003b:  ldstr      "http://loedata.legendsofequestria.com/data//"
+      IL_003b:  ldstr      "127.0.0.1:1034/data//"
       IL_0040:  ldloc.0
       IL_0041:  call       string [mscorlib]System.String::Concat(string,
                                                                   string)
@@ -7241,7 +7241,7 @@
                                                                        string)
       IL_0076:  br.s       IL_00ae
 
-      IL_0078:  ldstr      "http://loedata.legendsofequestria.com/data//"
+      IL_0078:  ldstr      "127.0.0.1:1034/data//"
       IL_007d:  ldloc.0
       IL_007e:  call       string [mscorlib]System.String::Concat(string,
                                                                   string)